interfaces/p2psentry/control.proto

```diff
@@ -21,8 +19,7 @@ message InboundMessage {
 
 message Forks {
   bytes genesis = 1;
-  repeated uint64 passed = 2;
-  uint64 next = 3;
+  repeated uint64 forks = 2;
```

Comment:
ForkID is uint32

Reply:
It's not ForkId, but fork block numbers. Sentry calculates ForkId and builds fork filter based on this information.